docs/how-to-monitor-minio-with-prometheus.md

```diff
@@ -10,7 +10,22 @@ This document explains how to setup Prometheus and configure it to scrape data f
 
 MinIO server release `RELEASE.2018-05-11T00-29-24Z` or later running. To get started with MinIO, refer [MinIO QuickStart Document](https://docs.min.io/docs/minio-quickstart-guide). Follow below steps to get started with MinIO monitoring using Prometheus.
 
-### 1. Download Prometheus
+### 1. Configure Prometheus in MinIO
+
+Prometheus is enabled and the authentication is set to "public" by default in MinIO configuration. For a secured access, update the prometheus config as follows
+
+``` json
+    "prometheus": {
+        "auth": "jwt",
+        "enable": true
+    }
+```
+
+To disable prometheus, set `enable` to `false`.
+
+To generate a bearer jwt token for an alias, use [mc](https://docs.min.io/docs/minio-client-quickstart-guide) like `mc admin prometheus jwt <alias>`.
```

Comment:
After setting jwt token, we give no information on how to access metrics with the token

Reply:
We talk about the dashboard in 4rth point here - `4. Start Prometheus`. This is more self-explanatory. I think we dont need to give them much details about the prometheus dashboard. That can be inconsistent with the dashboard updates. They can still explore/read about it in prometheus docs.